### ISSUE TITLE
add eslint-plugin-vue

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#258]
add the eslint-plugin-vue for linting of vue components

**Comment by kirlat** [comment]
In order to do Vue and regular JS file checks in one pass we have to switch to using eslint with "standard" config instead of a standalone standard package. The reason is `eslint-plugin-vue` is for eslint and thus cannot be integrated with standard package.

I did some tests, and it seems that eslint with standard config works fine with our files. Is it OK for me to make a switch? The other advantage could be that eslint is more widely supported and can be integrated with webpack, for example.

The other question I have is what check level for Vue shall we set as default. Shall it be "Essential", "Strongly Recommended", or "Recommended" (https://vuejs.org/v2/style-guide/)? What do you think?

**Comment by kirlat** [comment]
I've added a "standard' configuration to eslint and it works pretty much the same as the old good "standard" package. At least I was not able to notice any difference yet.

As for the vue plugin, even with "Essential" config that I've set it to it requires a lot of changes in the `.vue` files. Run on `*.vue` set of files in `components`, vue linter reports 284 errors! Many of them are not just spaces or things like keys for `v-for` (can be not that simple to fix either), but more complex things such as unused variables (should be very careful removing those - not sure how good liner is at that; there can be some tricky cases) or even computed prop that have side effects. I'm afraid it will take a lot of effort to fix them all at once and, what's worse, the whole thing will need to be retested.

So I think that probably it's better to postpone such global changes till the moment we actually change something in there, and this something be retested as part of this change workflow. So far, I can add a check of Vue files to `lint` run task only and with `--fix-dry-run` option that will not fix files automatically, but just output messages about issues found. For other build tasks I can set eslint to check `*.js` files only so far.

How does it sound? If acceptable, I will prepare a PR.
